### ISSUE TITLE
Group highlight for cards doesn't expand in launched sim

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/center-and-variability#585]
**Test device**
MacBook Air M1 chip

**Operating System**
14.0

**Browser**
Safari/Chrome

**Problem description**
For https://github.com/phetsims/qa/issues/993, the group highlight around the cards doesn't expand beyond the first card if balls have been kicked in studio prior to pressing Test. 

**Steps to reproduce**
1. In studio, on the Median screen, press the Kick 5 button
2. Press Test
3. Tab to the cards (or turn on interactive highlights and place your mouse there)

**Visuals**
<img width="1440" alt="Screenshot 2023-10-16 at 8 44 01 AM" src="https://github.com/phetsims/center-and-variability/assets/87318828/effddbb7-dd5d-4d37-97f2-9ba21b6d11c4">

**Comment by marlitas** [comment]
Fixed above. Over to @samreid or @matthew-blackman for review before marking for cherry-pick

**Comment by samreid** [comment]
I'll review

**Comment by samreid** [comment]
I reviewed the code and tested at runtime, and everything seems good. I could not recall why we had this guard before: `if ( this.cards && !isSettingPhetioStateProperty.value )`. Any ideas? In any case, this seems ready for cherry-pick.

**Comment by marlitas** [comment]
I also tried to look at the history and see what that guard was doing and couldn't figure it out. I tested in the state wrapper and couldn't find anything that indicated buggy behavior by removing that guard so my best guess was that we were maybe trying to be overly protective during state setting? 

Happy to test more situations where that might raise some alarms, but it was necessary to remove in order to implement the fix.
